index.html

```diff
@@ -20,19 +20,137 @@
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>2015</li>
+          </ul>
+          <p class="project-description-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="project-language-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="button-container">
+            <button class="see-project-button">See project</button>
+          </div> 
+        </section>
+      </article>
+      <article id="card-work-2" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 2_Availability.svg" alt="Project 2 Snapshot">
+        <section class="project-description-block">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="project-role-detail">
+            <li class="project-role-client ">CANOPY</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>Back End Dev</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>2015</li>
+          </ul>
+          <p class="project-description-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="project-language-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="button-container">
+            <button class="see-project-button">See project</button>
+          </div> 
+        </section>
+      </article>
+      <article id="card-work-3" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 3_Ballerina.svg" alt="Project 3 Snapshot">
+        <section class="project-description-block">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="project-role-detail">
+            <li class="project-role-client ">CANOPY</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>Back End Dev</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>2015</li>
+          </ul>
+          <p class="project-description-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="project-language-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="button-container">
+            <button class="see-project-button">See project</button>
+          </div> 
+        </section>
+      </article>
+      <article id="card-work-4" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 4_Art Printing.svg" alt="Project 4 Snapshot">
+        <section class="project-description-block">
```

Comment:
[OPTIONAL] I would suggest to use a div here rather than a section tag, as section is used to represent a generic part of the page and since it is just a division a div would be better here.